gitpod-io/gitpod
# jb-gw: consolidate welcome and recent screens

## Description


This PR consolidates the welcome and screen screens. It is particularly important with dynamic bundling of Gitpod plugin, since otherwise a user does not have a way to see all workspaces.

## Related Issue(s)

Fixes #

## How to test


- Download the plugin :https://plugins.jetbrains.com/plugin/download?rel=true&updateId=158209
- Install it manually in your gateway
- Restart gateway
- Check that both welcome and recent workspaces having the same flow now

## Release Notes

```release-note
NONE
```

## Documentation
